recipes/cuda-kat/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+class CudaKatConan(ConanFile):
+    name = "cuda-kat"
+    homepage = "https://github.com/eyalroz/cuda-kat"
+    description = "CUDA kernel author's tools"
+    topics = ("GPGPU", "CUDA", "cuda-kat", "header-only")
```

Comment:
```suggestion
    topics = ("gpgpu", "cuda", "cuda-kat", "header-only")
```
Avoid lower case, due sensitive-case search